charts/postgreslet/values.yaml

```diff
@@ -0,0 +1,439 @@
+# Default values for postgreslet.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: r.metal-stack.io/postgreslet
```

Comment:
Isn't the postgreslet located in ghcr.io? The metal-stack registry should not be used for public if possible.

Reply:
https://github.com/fi-ts/postgreslet/issues/237